src/quo/components/community/channel_action.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo.components.community.channel-action
+  (:require
+    [quo.components.community.style :as style]
+    [quo.components.counter.counter.view :as counter]
+    [quo.components.icon :as icons]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]))
+
+(defn view
+  [{:keys [big? color label counter-value icon on-press accessibility-label disabled?]}]
+  (let [theme                  (quo.theme/use-theme-value)
+        [pressed? set-pressed] (rn/use-state false)
+        on-press-in            (rn/use-callback #(set-pressed true))
```

Comment:
do we really need to wrap functions without dependencies in use-callback?

Reply:
it's to avoid re-creating the functions on every re-render